common/src/main/java/fr/openent/form/core/models/User.java

```diff
@@ -0,0 +1,55 @@
+package fr.openent.form.core.models;
+
+import io.vertx.core.json.JsonObject;
+
+import static fr.openent.form.core.constants.Fields.*;
+
+public class User implements Model<User> {
```

Comment:
On va préférer l'étendre de UserInfo, pour éviter de réécrire ce qui existe déjà ;)

Ex:
https://github.com/OPEN-ENT-NG/minibadge/blob/dev/src/main/java/fr/openent/minibadge/model/User.java

Reply:
C'est pas un object User de ce type là, c'est un User qui modélise les infos dans la table Users du système de partage. A la limite j'peux le renommer and ShareUser ou un truc comme ça si tu veux :)